ext/posix/posix.c

```diff
@@ -1208,7 +1208,11 @@ PHP_FUNCTION(posix_pathconf)
 		Z_PARAM_LONG(name);
 	ZEND_PARSE_PARAMETERS_END();
 
-	if (path_len == 0 || php_check_open_basedir(path)) {
+	if (path_len == 0) {
+		zend_value_error("Empty path supplied");
```

Comment:
```suggestion
		zend_argument_value_error(1, "cannot be empty");
```
Is the standard way as this will also indicate what argument and it's formal param name from the stubs